docs/mosquitto.md

```diff
@@ -0,0 +1,27 @@
+## Description
+**This script was broken since packages are not available for Debian Stretch, but we have implemented an workaround that make sure this will run as intended on HASSbian Stretch.**  
```

Comment:
This is a tad out of date now ;)

Reply:
I was not, when I first made this :), corrected now